docs/os/armbian-build.md

```diff
@@ -35,8 +35,22 @@ In Linux you can directly run `make`, while in Windows PowerShell you need to ru
 
 * The virtual machine will be configured running on 4 cpu cores and 8GB memory. You can adjust these settings in the `build.sh` script.
 
+### Configure build options
+
+The build itself and the initial configuration of the Base image (e.g. hostname, Bitcoin network or Wifi credentials) can be configured within the configuration file [`armbian/base/build/build.conf`](https://github.com/digitalbitbox/bitbox-base/blob/master/armbian/base/build/build.conf). 
+Options are described and set to default values.
```

Comment:
(Optional) I think I understand what we're going for in this part of the text, but perhaps a more clear way to word it would be:
```
The available options for the build are described in `build.conf`.
```

Alternatively, maybe it's clear enough to link to `build.conf` above, and we can drop the sentence on L41?